run.py

```diff
@@ -1,21 +1,26 @@
+import os
+import logging
 from json import loads
 
 import structlog
+from flask import Flask, _app_ctx_stack
 from flask_cors import CORS
-from ras_common_utils.ras_config import ras_config
-from ras_common_utils.ras_config.flask_extended import Flask
-from ras_common_utils.ras_database.ras_database import RasDatabase
-from ras_common_utils.ras_logger.ras_logger import configure_logger
 from retrying import retry, RetryError
+from sqlalchemy import create_engine
 from sqlalchemy.exc import DatabaseError
+from sqlalchemy.orm import scoped_session, sessionmaker
 
-logger = structlog.get_logger()
+from logger_config import logger_initial_config
 
 
-def create_app(config):
+logger = structlog.wrap_logger(logging.getLogger(__name__))
+
+
+def create_app(config=None):
     # create and configure the Flask app
     app = Flask(__name__)
-    app.config.from_ras_config(config)
+    app_config = f"config.{config or os.environ.get('APP_SETTINGS', 'Config')}"
```

Comment:
is that "f" supposed to be there?

Reply:
Yep. It's an f-string style for formatting the config string in place.